ci-operator/step-registry/openshift/e2e/test/capabilities-check/openshift-e2e-test-capabilities-check-commands.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+cp -Lrvf ${KUBECONFIG} /tmp/kubeconfig && export KUBECONFIG=/tmp/kubeconfig
+oc new-project e2e-test-capabilities-check
+
+if [[ "${DELETE_MC}" == "true" ]]; then
+    oc delete mc 99-worker-generated-crio-capabilities
+    # need to wait for the changes to roll out
+    # for debug purposes only
+    echo "waiting for the changes to roll out"
+    oc wait --for=condition=Updating=false mcp/worker --timeout=600s
+    echo "done waiting"
+fi
+
+# create a fedora pod and get the capabilities enabled in the pod
+oc run fedora-pod --image fedora --restart Never --command -- sleep 1000
+capabilities=$(oc rsh fedora-pod capsh --print)
+# for debug purposes only, will remove once tests are passing
+echo "capabilities are ${capabilities}"
+
+# get the capabilities MCs available
+mcCaps=$(oc get mc | grep 99-worker-generated-crio-capabilities)
```

Comment:
```suggestion
mcCaps=$(oc get mc/99-worker-generated-crio-capabilities -o name; true)
```
due to `set -o errexit` this would not exit immediately

Reply:
got it, thank you!